jdaviz/configs/cubeviz/plugins/mixins.py

```diff
@@ -89,6 +89,12 @@ def slice_values(self):
         take_inds = [2, 1, 0]
         take_inds.remove(self.slice_index)
         converted_axis = np.array([])
+
+        # Retrieve display units
```

Comment:
just confirming, this was moved because the slice display units don't depend on layer?

Reply:
right, I moved this above the for-loop since it isn't needed to be called repeatedly